src/pages/Flow/Details-Tile.vue

```diff
@@ -70,6 +70,28 @@ export default {
           return obj
         }, {})
     },
+    runConfigDisplayFields() {
+      if (!this.flow.run_config || !this.flow.run_config.type) return []
+
+      let fields = ['type']
+      if (this.flow.run_config.type == 'LocalRun') {
+        fields.push('working_dir')
+      } else if (this.flow.run_config.type == 'DockerRun') {
+        fields.push('image')
+      } else if (this.flow.run_config.type == 'KubernetesRun') {
+        fields.push(
+          'image',
+          'job_template_path',
+          'cpu_request',
+          'cpu_limit',
+          'memory_request',
+          'memory_limit'
+        )
+      } else if (this.flow.run_config.type == 'ECSRun') {
+        fields.push('image', 'task_definition_path', 'cpu', 'memory')
+      }
+      return fields.filter(field => this.flow.run_config[field] != null)
```

Comment:
```suggestion
      return fields.filter(field => field in this.flow.run_config)
```
Let's use this instead because there are a lot of edge cases when using `!= null`; this will check that the property exists. If we specifically need to filter out `null` (rather than `undefined`), we could instead us `!== null`

Reply:
I'm trying to filter out fields that are missing or are explicitly set as `null` (but do want to keep other false-y values like `""` or `0`). Does this do that? I am not a JS dev, pulled this from this [stackoverflow answer](https://stackoverflow.com/a/15992131), which seemed to imply that `undefined == null` is true.